command/forceleave/forceleave.go

```diff
@@ -47,10 +52,20 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
-	err = client.Agent().ForceLeave(nodes[0])
-	if err != nil {
-		c.UI.Error(fmt.Sprintf("Error force leaving: %s", err))
-		return 1
+	switch c.prune {
```

Comment:
It seems more natural to use `if c.prune { ... } else { ... }` instead of a boolean switch statement here.

Also you can move the common error handling outside of the conditional block, like:
```
if c.prune {
   err = client.Agent().ForceLeavePrune(...)
} else {
  err = client.Agent().ForceLeave(...)
}
if err != nil {
   ...
}
```

Reply:
When you say natural, do you mean easier to read? Or what do you mean by natural?

Not against your suggestion, but I'm wondering which is more idiomatic but also user friendly? I've been going back and forth on which is better, why, and when to use a switch versus an if/else. Feedback would be amazing ❤️ 
(Perhaps it's more like the rest of Consul code to be an if/else 🤷‍♀ ?)